src/partials/marketStats/marketStats.vue

```diff
@@ -34,35 +34,22 @@
  * Importing libraries
  */
 import { mapState } from 'vuex'
-import polling from '../../functions/polling'
-
-/*
- * Creating polling instance
- */
-const poll = polling()
+import pollAction from '../../mixins/pollAction'
 
 /**
  * Export component
  */
 export default {
+  mixins: [pollAction('_marketStats/get')],
+
   /*
    * Computed properties
    */
   computed: mapState('_marketStats', [
     'priceChf',
     'marketCapChf',
     'priceBtc'
-  ]),
-
-  /*
-   * Create poll function for market-stats
-   */
-  mounted: function () {
-    return poll.fetch.call(this, '_marketStats/get')
-  },
-  destroyed: function () {
-    return poll.close('blocks/getLatestBlocks')
```

Comment:
As I see, action name in `destroyed` should be the same as in `mounted`.

> I think the pattern, that each screen needs to this.unsubscribe() in beforeDestroy is error prone. https://github.com/aeternity/aepp-blockchain-explorer/pull/10#issuecomment-379461890

@dadaphl, you were right.